admin_pages/transactions/EE_Admin_Transactions_List_Table.class.php

```diff
@@ -148,9 +148,9 @@ protected function _get_table_filters()
      */
     protected function _add_view_counts()
     {
-        $this->_views['all']['count'] = $this->_admin_page->get_transactions($this->_per_page, true, 'all');
-        $this->_views['abandoned']['count'] = $this->_admin_page->get_transactions($this->_per_page, true, 'abandoned');
-        $this->_views['failed']['count'] = $this->_admin_page->get_transactions($this->_per_page, true, 'failed');
+        foreach ($this->_views as $view) {
+            $this->_views[ $view['slug'] ]['count'] = $this->_admin_page->get_transactions($this->_per_page, true, $view['slug']);
+        }
```

Comment:
should go through and make this change to ALL of our list tables (not as part of this PR though)